src/restful/routes/authRouter.js

```diff
@@ -13,5 +13,7 @@ router.post("/verify", AuthController.verifyOtp);
 router.post("/reset", AuthController.resetUserPassword);
 router.post("/username", protect, AuthController.createUsername);
 router.get("/user/:user_id", AuthController.getPublicUser);
+router.delete("/users/:user_id", protect, AuthController.deleteUser); // New route added for deleting user account
```

Comment:
I noticed the route for deleting users, but it seems that the corresponding controllers are missing.

Reply:
explain what you mean by controlers are missing

Reply:
Byes we have, its already on my local repository, I think it got removed mistakenly! it is fixed niw, thanks for the feedback